bin/latex.py

```diff
@@ -144,20 +141,50 @@ def get_environment():
     return env
 
 
+def build_latex_pdf(builddir, tex_path, problem_path=None):
+    env = get_environment()
+    run_once = getattr(config.args, '1', False)
+    watch = getattr(config.args, 'watch', False)
+    ret = util.exec_command(
+        ['latexmk', '-cd', '-g', '-pdf',
+         '-pdflatex=pdflatex -interaction=nonstopmode -halt-on-error']
+        + (['-pvc'] if watch else [])
+        + (['-e', '$max_repeat=1'] if run_once else [])
+        + [f'-output-directory={builddir}', tex_path],
+        0,
```

Comment:
Could you comment what these variables mean? (I know it's copy paste, but I forgot :P )

Reply:
And with "these variables" you mean the parameters `0` and `False` passed to `util.exec_command`? I "commented" them by passing them as keyword arguments.